auto-cpufreq.py

```diff
@@ -210,7 +210,7 @@ def set_performance():
     print("\nTotal CPU usage:", cpuload, "%")
     print("Total system load:", load1m, "\n")
 
-    # conditions for setting turbo in performance
+    # conditions for setting turbo in performance for future using
     if load1m > 1:
```

Comment:
What did you add "for future using"?

Reply:
This comes with the first commit of that PR, undoing the change.